src/clinical/clinical-entities.ts

```diff
@@ -36,4 +34,18 @@ export interface Sample {
   submitterId: string;
 }
 
+export interface Treatment {
+  clinicalInfo?: ClinicalInfo;
+  chemotherapy: Array<Therapy>;
+  hormoneTherapy: Array<Therapy>;
+}
+
+export interface Therapy {
+  clinicalInfo?: ClinicalInfo;
+}
+
+export interface ClinicalInfo {
+  [field: string]: string | number;
```

Comment:
this could also be boolean | undefined

Reply:
(:+1: ) true, I just copy pasta'd what was already their for all clinicalInfo, will update..

Reply:
yes but you also generalized this, before it was specific to donor and specimen, which according to the schema didn't have boolean or other stuff

Reply:
do we want to keep that information or is it ok to generalize it?

Reply:
it's ok